.gitlab-ci.yml

```diff
@@ -180,7 +180,7 @@ opam-boot:
       - .opamcache
     key: "main-$CACHEKEY"
   variables:
-    EXTRA_OPAM: "$COQIDE_OPAM ocamlgraph $ELPI_OPAM"
+    EXTRA_OPAM: "$COQIDE_OPAM ocamlgraph $ELPI_OPAM jbuilder"
```

Comment:
What's going on?

Reply:
I am adding indeed Dune to the base image; IMHO not a big deal; I guess I'll get the Docker PR merged before this one so it won't matter.

Reply:
Reified Dune changes to their own commit.